src/components/existing-request/existing-request-search.vue

```diff
@@ -66,7 +66,7 @@
                             filled
                             id="phone-number-text-field"
                             label="Applicant's Phone Number"
-                            hint="Example: 555-555-5555"
+                            hint="Example: 555-555-5555 or 555 555 5555"
```

Comment:
Check with a PO whether this is OK. I've been see people just typing in `7786983639` and that doesn't validate any more (which may be correct -- I'm not saying it isn't -- just check to approve this change).

Reply:
For sure, this is a really good point.
I think my main goal here was to protect what appears to be broken logic in the api, but we can run this by Linda.
I could also adjust the regex to allow 10 straight characters.

Reply:
OK, I'll leave this with you to determine and adjust as needed.

This all looks like the right things to do. Well done. GTG

Reply:
^^^ it was decided to not do phone number validation in the UI.

There is a ticket to fix phone number validation in the namex API.

Reply:
Yes! Sorry, ticket #6950